Tella/Data/Networking/Models/UwaziCheckURLResult.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  UwaziCheckURLResult.swift
+//  Tella
+//
+//  Created by Robert Shrestha on 5/22/23.
+//  Copyright © 2023 INTERNEWS. All rights reserved.
+//
+
+import Foundation
+
+struct UwaziCheckURLResult: Codable {
```

Comment:
@dhekra-rouatbi Both `siteName` and `id` is used to I do not think we need to implement a DTO. What do you think ?

Reply:
@rob-horizontal-iOS I think It's better to work with DTO in our case, because the objects from API are composed from many other objects in the most of the cases (I think only in this case we have 2 attributes)
If there is changes in the data source do not effect (necessarily) our domain objects

Could you apply the mapping to the structs that you create and avoid to use DTO in the view model and try to use domain model